### DISCUSSION TITLE
Images are so large that can'y load!!!

### DISCUSSION THREAD

**Discussion by SnoopyXI** [Project-MONAI/MONAI#7400] [General]
**Describe the bug**
My images are 30000x10000, which are so large that can't load by Pillow, but opencv can load images. So I wanna know how to use cv2 as reader in LoadImage function. Or is there any better solution?

**Error**:
PIL.Image.DecompressionBombError: Image size (606563352 pixels) exceeds limit of 178956970 pixels, could be decompression bomb DOS attack.

**Comment by KumoLiu** [comment]
Hi @SnoopyXI, you can write your own reader inherit from `ImageReader` and register into `LoadImage`.
https://github.com/Project-MONAI/MONAI/blob/25f190148c7c3e1326e55e296ef7c374903b3c52/monai/data/image_reader.py#L57

https://github.com/Project-MONAI/MONAI/blob/25f190148c7c3e1326e55e296ef7c374903b3c52/monai/transforms/io/array.py#L217
